### ISSUE TITLE
callback output in v2.1.5

### ISSUE THREAD

**Issue by ironv** [keras-team/keras#9649]
I have a callback class which inherits from `keras.callbacks.Callback`.   In the `on_train_begin` method in that class I have added four new metrics to `self.params['metrics']`, which compute the AUC and AP on training and validation data.  The computation is implemented in `on_epoch_end` and the logs dictionary is assigned the respective values.

When this was running in Keras 2.1.3, at the end of each epoch, the output would look like
`     ==============    400s  290ms/step - loss: 0.3712 - tr_auc: 0.8012 ....   va_ap: 0.912`

I just updated to Keras 2.1.5 and the output only shows
`     ==============    400s  290ms/step - loss: 0.3712`

I know that that all the metrics that I am computing are in fact being computed because (i) an early stopping callback based on average precision on validation is working fine.  (ii) able to use model.history after the run is completed to plot all the metrics computed above at the end of each epoch.

I can obviously print it from the class but has something changed where I would need to explicitly print that information?  What is the right way of outputting that information at the end of each epoch?  Thx.
